### ISSUE TITLE
Please support minimatch options 

### ISSUE THREAD

**Issue by nefrob** [actions/labeler#434]
Ex. I want to match dotfiles by adding `dot: true` option to my glob pattern to minimatch.

**Comment by kachkaev** [comment]
Possible duplicate of https://github.com/actions/labeler/issues/135

**Comment by nefrob** [comment]
@kachkaev yes it appears so. Closing, but it would be great to get the above PR approved and merged. Thanks.
